data/levels/revenge_in_redmond/intro.txt

```diff
@@ -0,0 +1,32 @@
+(supertux-text			
+  (background	"retro/intro.jpg")
+  (music	"music/retro/theme_old.music")
+  (text "-A Blast From The Past!
+ Or was it a dream?
+
+	Tux and Gown were having a nice picnic
+	when suddenly Gown is beamed away!!!

```

Comment:
`was beamed away` I think.

Reply:
funny enough this is a typo in the ST 0.0.6 version which I copyed this line from 1:1 but I think fixing it makes sense here no mattter what

Reply:
For me it sounds good both ways so it can be kept as is.

Reply:
However, I think it needs commas around `when suddenly`.

Reply:
However, I think it needs commas around `when suddenly`.

Reply:
Not so sure on te commas here. Isn't it that you only use commas before and after parts of the sentence that are not essential to its meaning? The fact Gown is beamed away during the picnic seems rather essential in this case.

Reply:
Since I got no further reaction here, nor is it anything major, I consider this resolved